<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fifteen</class>
 <widget class="QMainWindow" name="Fifteen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пятнашки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/icon_fifteen.png</normaloff>:/image/icon_fifteen.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>381</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn10">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn15">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn8">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn5">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn14">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>14</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn4">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn6">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn13">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>13</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn11">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>11</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn9">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn7">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btn12">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>12</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn3">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_shufle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>11111</width>
      <height>1111</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(68, 68, 68);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>СТАРТ</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_time">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(135, 135, 135);</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>167</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>36</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(150, 150, 150);</string>
         </property>
         <property name="text">
          <string>ШАГИ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="count">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>11111</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>36</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(150, 150, 150);
border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="levelBox">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Классическая игра</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ночной фонарь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Весенний кот</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Летний кот</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Чизкейк</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Кролик</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Корги</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Суши</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Шрек</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Рим</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_manual">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>460</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
color: rgb(0, 0, 0);
image: url(://image/manual.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="shortcut">
     <string>F1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
color: rgb(0, 0, 0);
image: url(://image/exit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_records">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>460</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
selection-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
image: url(://image/records.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
